.github/release-drafter.yml

```diff
@@ -0,0 +1,29 @@
+categories:
+  - title: ":boom: Breaking Changes"
+    label: "breaking"
+  - title: ":rocket: Features"
+    label: "enhancement"
+  - title: ":fire: Removals and Deprecations"
+    label: "removal"
+  - title: ":beetle: Fixes"
+    label: "bug"
```

Comment:
```suggestion
    labels:
      - "bug"
      - "fix"
```

Reply:
Are you sure this is necessary? We should avoid similar labels to faciltate issue and PR triage. As admin we are choosing the label. I don't understand the need to have different wording for the same label type.

For information the labels have been already created in this repository.

Reply:
Oh yes good !
I thought it was the text included in the PR title :sweat_smile: 

So it should have one everywhere sure :+1: